src/js/data/names.js

```diff
@@ -3436,7 +3436,7 @@ let first = {
         ["A.C.", 5],
         ["A.J.", 63],
         ["A.W.", 64],
-        ["ATorri", 65],
```

Comment:
Didn't notice that comment, sorry. How would you like me to proceed with these kinds of batch name updates, then? Pull request tools/names.js and have you re-run?